### ISSUE TITLE
Laravel 5.5 Error on migration

### ISSUE THREAD

**Issue by gesicmerrybeard** [markhilton/monolog-mysql#1]
I did follow all the installation steps and get following error when im at the step:

`php artisan migrate` 

> [Symfony\Component\Debug\Exception\FatalThrowableError] Class '' not found

**Comment by dvdhoek** [comment]
Boot should be changed to 

$this->loadMigrationsFrom(__DIR__.'/path/to/migrations');

If you rename the migration file everything works fine.

**Comment by d-nekrasov** [comment]
I did follow all the installation steps and get following error when im at the step:

> In Migrator.php line 417:
> Class '' not found




**I solved this problem. The problem in the name of migration**

**Comment by dmytro-kuchura** [comment]
Helllo, need before migration name file write date, for examle 2017_12_20_085205_create_logs_table.php

For me it's work

**Comment by askme-gpt** [comment]
Yes ,you are right , I solved this problem . we need to add d date and random number before the table .
